package/MDAnalysis/auxiliary/base.py

```diff
@@ -43,13 +43,15 @@
 import numbers
 import math
 import warnings
+from typing import Union, Optional, Dict
 
 import numpy as np
 
 from ..lib.util import asiterable, anyopen
+from .. import units
 
 from . import _AUXREADERS
-
+from .core import auxreader
```

Comment:
```suggestion
from .core import auxreader

```
2 lines between imports and class def